gdal/port/cpl_http.cpp

```diff
@@ -380,7 +412,9 @@ double CPLHTTPGetNewRetryDelay(int response_code, double dfOldDelay)
  * @return a CPLHTTPResult* structure that must be freed by
  * CPLHTTPDestroyResult(), or NULL if libcurl support is disabled
  */
-CPLHTTPResult *CPLHTTPFetch( const char *pszURL, char **papszOptions )
+CPLHTTPResult *CPLHTTPFetch( const char *pszURL, char **papszOptions,
```

Comment:
Can we make this a CPLHTTPFetchEx() function so as to avoid changing the C API (not sure if there are many external users, but just in case) ?

Reply:
Fixed